gradle-plugin/src/main/java/com/google/cloud/tools/dependencies/gradle/LinkageCheckTask.java

```diff
@@ -130,22 +146,24 @@ private boolean findLinkageErrors(Configuration configuration) throws IOExceptio
       // TODO(suztomo): Specify correct entry points if reportOnlyReachable is true.
       LinkageChecker linkageChecker = LinkageChecker.create(classPath, classPath, exclusionFile);
 
-      ImmutableSetMultimap<SymbolProblem, ClassFile> symbolProblems =
-          linkageChecker.findSymbolProblems();
+      ImmutableSet<LinkageProblem> linkageProblems = linkageChecker.findLinkageProblems();
 
-      int errorCount = symbolProblems.keySet().size();
+      LinkageProblemCauseAnnotator.annotate(classPathResult, linkageProblems);
```

Comment:
This is the goal of this PR. To call this method, we need to construct the first argument (ClassPathResult) from the data provided by Gradle.